source/agora/cli/client/GenTxProcess.d

```diff
@@ -0,0 +1,181 @@
+/*******************************************************************************
+
+    A tool which periodically generates a set of valid transactions
+    and sends them to one node. The interval at which the transactions
+    are sent and the number of transactions can be specified.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.client.GenTxProcess;
+
+import agora.api.FullNode;
+import agora.client.Result;
+import agora.common.Amount;
+import agora.common.crypto.Key;
+import agora.common.Hash;
+import agora.common.Task;
+import agora.common.Types;
+import agora.consensus.data.Block;
+import agora.consensus.data.Transaction;
+import agora.consensus.data.genesis.Test;
+import agora.utils.PrettyPrinter;
+import agora.utils.Test;
+
+import std.algorithm;
+import std.conv;
+import std.format;
+import std.getopt;
+import std.range;
+import std.stdio;
+
+import core.time;
+
+import vibe.core.core;
+
+public alias APIMaker = API delegate (string address);
+
+/// Option required to generate and send transactions
+private struct GenTxOption
+{
+    /// IP address of node
+    public string host;
+
+    /// Port of node
+    public ushort port;
+
+    /// Interval of sending transactions
+    public Duration interval = 5.seconds;
+
+    /// Number of transactions sent at once (1 to 8)
+    public uint count = 1;
+
+    /// dump output option
+    public bool dump;
+}
+
+/// Parse the command-line arguments of gentx
+public GetoptResult parseGenTxOption (ref GenTxOption op, string[] args)
+{
+    return getopt(
+        args,
+        "ip|i",
+            "IP address of node",
+            &op.host,
+
+        "port|p",
+            "Port of node",
+            &op.port,
+
+        "interval|t",
+            "Interval of sending transactions (default: 5 seconds)",
+            (string value) { op.interval = value.to!int.seconds; },
+
+        "count|c",
+            "Number of transactions sent at once (default: 1)",
+            &op.count,
+
+        "dump|o",
+            "Dump output option",
+            &op.dump
+            );
+}
+
+/// Print help
+public void printGenTxHelp (ref string[] outputs)
```

Comment:
`getopt` gives you this for free, check the documentation.
Although it's not a big deal (aka you don't have to change it now), because the current version is 👍 and works.